src/Microsoft.AspNetCore.Hosting.Abstractions/WebHostDefaults.cs

```diff
@@ -12,7 +12,7 @@ public static class WebHostDefaults
         public static readonly string WebRootKey = "webroot";
         public static readonly string CaptureStartupErrorsKey = "captureStartupErrors";
         public static readonly string ServerUrlsKey = "server.urls";
-        public static readonly string ApplicationBaseKey = "applicationBase";
```

Comment:
(It's sort of a mental note to myself to update publish-iis to account for this rename)